answer_files/81/Autounattend.xml

```diff
@@ -131,83 +131,11 @@
                     <RequiresUserInput>true</RequiresUserInput>
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
-                    <CommandLine>cmd.exe /c winrm quickconfig -q</CommandLine>
-                    <Description>winrm quickconfig -q</Description>
-                    <Order>3</Order>
-                    <RequiresUserInput>true</RequiresUserInput>
-                </SynchronousCommand>
-                <SynchronousCommand wcm:action="add">
-                    <CommandLine>cmd.exe /c winrm quickconfig -transport:http</CommandLine>
-                    <Description>winrm quickconfig -transport:http</Description>
-                    <Order>4</Order>
-                    <RequiresUserInput>true</RequiresUserInput>
-                </SynchronousCommand>
-                <SynchronousCommand wcm:action="add">
-                    <CommandLine>cmd.exe /c winrm set winrm/config @{MaxTimeoutms="1800000"}</CommandLine>
-                    <Description>Win RM MaxTimoutms</Description>
+                    <CommandLine>C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -File a:\disable-winrm.ps1</CommandLine>
```

Comment:
I think it should be `a:\enable-winrm.ps1` ? 🤔

Reply:
Hi. I don't think so. Essentially I made this file to be equal to the `Autounattend.xml` file of Windows **7**. The third `<SynchronousCommand>` (for Windows 7) is to **disable** WinRM (not sure why this is needed, assuming that WinRM is not enabled by default anyway). The _last_ `<SynchronousCommand>` then installs Windows updates (and `win-updates.ps1` will then enable WinRM). This order of `<SynchronousCommand>`s will make sure that packer will only get WinRM-connectivity after the updates have finished installing, and not before. Only then will it continue with other post-processing steps like installing Guest additions.
However, I'd like to note that I'm not deeply familiar with packer, these templates, or Windows-related things. I've just hacked things so that they work, and tried to make Windows 7 and 8.1 more similar in the process.